style.css

```diff
@@ -487,3 +487,104 @@ hr {
   color: #fff;
   margin-top: 15px;
 }
+
+.contact {
+  width: 100%;
+}
+
+.contact h2 {
+  width: 85%;
+  margin: 0 auto;
+  margin-top: 100px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 900;
+  font-size: 34px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  height: 216px;
+}
+
+.ip-type {
+  display: flex;
+  flex-direction: column;
+  width: 87%;
+  margin: 0 auto;
+  position: relative;
+  top: 50px;
+}
+
+.email {
+  border: none;
+  border-bottom: 1px solid #dfe1e6;
+  height: 30px;
+  margin-bottom: 25px;
+}
+
+.Name {
+  border: none;
+  border-bottom: 1px solid #dfe1e6;
+  height: 30px;
+  margin-bottom: 25px;
+}
+
+.t1 {
+  height: 114px;
+  background: #fbfbf7;
+  border: none;
+  text-justify: top;
+  font-weight: bolder;
+}
+
+.get {
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are not styled as shown in the Figma template, please implement this so your design can be as close to the provided design as possible